File: 0873/873.go
func lengthOfLongestFibonacciSubseq(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := lengthOfLongestFibonacciSubseq(root.Left)
	r := lengthOfLongestFibonacciSubseq(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0873/873.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var lengthOfLongestFibonacciSubseq = function(root) {
    if (!root) return 0;
    return 1 + Math.max(lengthOfLongestFibonacciSubseq(root.left), lengthOfLongestFibonacciSubseq(root.right));
};